app/services/booking_responder.rb

```diff
@@ -84,4 +86,17 @@ def mark_banned_visitors
       visitor.update! banned: true
     end
   end
+
+  def create_message
+    return nil if booking_response.message_body.blank?
+
+    Message.create!(body: booking_response.message_body,
+                    user: booking_response.user,
+                    visit: visit,
+                    visit_state_change: visit.last_visit_state)
```

Comment:
When creating a message within the booking responder it is always related to the last visit state change. Other messages that aren't created via the booking responder won't be using this method, to me it is pretty obvious so don't know what to change it to. Should we leave it as it is and wait for the events work?